src/components/shared/transactionTypeFigure/index.js

```diff
@@ -8,7 +9,7 @@ const TransactionTypeFigure = ({
   moduleAssetId, address, avatarSize = 40, className = '', icon,
 }) => {
   const renderAvatar = () => {
-    if (regex.address.test(address)) {
+    if (validateAddress(tokenMap.LSK.key, address) === 0) {
```

Comment:
Wouldn't it be better if we return a boolean from `validateAddress()`?

Reply:
There are three states:
-1: address blank. This is useful for not displaying an error, but also not enabling the submit/continue button
0: address valid
1: address invalid

Reply:
If we want to use codes, I think you can use the following:
1: valid (no need to check against 1, its a truthy value)
0: invalid (no need to check against 0, its a falsy value)
-1: (invalid and) empty string (-1 is truthy but you would need to specifically check against it) 

simplifies our assertions in the components